script/github-actions/daily-product-scan/tests/mocks/applications/app-2/app-2b/containers/WizardContainer.jsx

```diff
@@ -28,7 +28,7 @@ const WizardContainer = props => {
           28-8832
         </h1>
         <p>
-          Equal to VA Form 28-8832 (Education/Vocational Counseling Application)
+          Equal to VA Form 25-8832 (Education/Vocational Counseling Application)
```

Comment:
If you remove this change, I don't think you'll need QA standards to approve this PR. I believe this "daily-product-scan" folder gets updated automatically from the content build repo `registry.json` - which would also require the same changes as the `manifest.json` file.

Reply:
Thank you @Mottie this is very helpful. To clarify my understanding we can remove the above change and the changes in [this PR](https://github.com/department-of-veterans-affairs/content-build/pull/1670/files) should make their way back to here? Does this sounds correct?